getting-started/best-practices.mdx

```diff
@@ -11,34 +11,56 @@ Rive is built to efficiently play interactive graphics in the editor and at runt
+
+##### **Image Dimensions for WebGL runtimes**
+
+For WebGL runtimes, image memory is allocated in powers of two. A 512×512 image fits efficiently, but a 513×513 image will use the memory of a 1024×1024 texture—quadrupling the memory usage for just one extra pixel per side.
+
+#### Raster Image Compression
 
 Compression involves reducing the file size by employing various algorithms to discard some image data. You can compress images directly from the Rive editor, which means you maintain the original image but compress the images for runtime use. If the asset is embedded in the animation, this will reduce the file size of the exported riv.
 
-#### Other Considerations
+For the smallest image file size and best performance, we recommend exporting assets using the WebP Format.
+
+#### Vector Images
 
-- If you’re importing assets for reference only, ensure they are deleted before exporting the .riv file to ensure the asset doesn’t get bundled in.
-- Depending on your target device/platform, it may make sense to have multiple different raster assets. For example, use smaller images when targeting mobile vs. desktop.
-- See our docs on [Loading Assets](/runtimes/loading-assets), to dynamically load in assets at runtime, instead of embedding the files in the exported `.riv` binary.
+Be efficient with the number of vertices in your vector image. While a few extra vertices won’t have much of an impact, thousands might. Be especially careful when importing vectors that were generated with AI, converted from raster images, or were created in drawing apps.
 
 ### Importing Lottie Files
 
 While Rive provides a Lottie converter for easy .riv file export, recreating graphics and animations directly in Rive often results in significantly smaller file sizes. If you're importing a Lottie file, you can further reduce the .riv file size by converting images from PNG to WebP and choosing to export only the necessary glyphs for font files. Additionally, you can [load assets out of band](#out-of-band-assets) to reuse fonts and images across multiple .riv files, optimizing storage.
 
 Working directly in Rive is generally preferable as it allows for file optimization based on your animation requirements. For instance, using bones and constraints to create rigs will result in fewer keys in the animation compared to converting directly from Lottie.
 
+### Artboard Considerations
+
+#### Clipped Artboards
+
+Clipping artboards is generally fine, but if you're experiencing performance issues, it's worth minimizing their use. Clipping can be computationally expensive, as the renderer must evaluate every object—including nested artboards—to determine pixel visibility. Instead, consider applying clips to specific objects or groups within the artboard.
+
+You can also safely remove the clip from the main artboard itself, since nothing outside the Rive instance will be rendered at runtime.
```

Comment:
Not true for all the runtimes! But, for now it's more common for this to be the case